Bindings/Python/Makefile.in

```diff
@@ -50,6 +50,7 @@ $(PYTHON_API): brlapi.auto.c $(API_HDRS) | $(INSTALLED_FILES) brlapi
 	set -- $(V_setup) build --build-temp .; \
 	[ "@host_os@" != "mingw32" ] || set -- "$${@}" --compiler mingw32; \
 	"$(PYTHON)" ./setup.py "$${@}"
+	[ "@host_os@" != "mingw32" ] || "$(PYTHON)" ./setup.py $(V_setup) bdist_egg --skip-build
```

Comment:
I don't know what is going on here exactly. I guess we don't match the mingw32 in your msys-2 build? Also, the current BRLTTY release for Windows contains a Python wheel, so it would be nice if we create a wheel as well instead of an egg.